File: 4_Simple_Calculator/main.c
#include<stdio.h>

void calculator(int x, int y)
{ 
  int choice;
  printf("Enter 1 for Addition\nEnter 2 for Subtraction\nEnter 3 for Multiplication\nEnter 4 for Division\n");
  printf("Enter Your Choice: ");
  scanf("%d", &choice);
  switch(choice)
  {
    case 1:
    {   
        printf("The Addition of a and b is: ");
        printf("%d", x+y);
        break;
    }
    case 2:
    {
        printf("The Subtraction of a and b is: ");
        printf("%d", x-y);
        break;
    }
    case 3:
    {
        printf("The Multiplication of a and b is: ");
        printf("%d", x*y);
        break;
    }
    case 4:
    {
        printf("The Divsion of a and b is: ");
        printf("%d", x/y);
        break;
    }
    default:
    {
        printf("Please Enter a Valid Choice!");
    }
  }
}

int main()
{   int a,b,c;
    printf("Enter the value of a: ");
    scanf("%d", &a);
    printf("Enter the value of b: ");
    scanf("%d", &b);
    calculator(a,b);
    return 0;
}

File: 4_Simple_Calculator/main.py
def calculator(x,y):
    choice = int(input("\nEnter 1 for Addition\nEnter 2 for Subtraction\nEnter 3 for Multiplication\nEnter 4 for Division\nPlease Enter Your Choice: "))
    match choice:
        case 1:
            print("\nThe Addition of a and b is: ",a+b)
        case 2:
            print("\nThe Subtraction of a and b is: ", a-b)
        case 3:
            print("\nThe Multiplication of a and b is: ", a*b)
        case 4:
            print("\nThe Division of a and b is: ", a/b)
        case _:
            print("\nPlease enter a valid choice! ")
    

if __name__ == "__main__":
    a = int(input("Enter the value of a: "))
    b = int(input("Enter the value of b: "))
    calculator(a,b)